docs/journal/01-cold-storage.md

```diff
@@ -1,74 +1,90 @@
-# Journal Entry #1
-_Capturing event data and saving it in its raw format to cold storage_
+# Journal 1: Event Ingestion and Cold Storage
 
-:pencil2: working draft
+This journal entry covers the most basic end-to-end scenario in IoT: Getting events into the system and putting them into long-term storage. In terms of the logical architecture, it maps to the highlighted components shown here:
 
-> The purpose of this phase is to determine an approach to ingesting data, performing the simplest of processing, and saving it for subsequent analysis. Because the customer requires all event data to be stored indefinitely, the volume of data held in cold storage could become very large. Cold storage must therefore be inexpensive.
+![plan for this first milestone](media/01-cold-storage/logical-architecture-for-milestone01.png)
 
-# Fabrikam's Perspective
-Fabrikam's engineering team has very little experience with high-scale event-oriented systems. They want to make sure that they are [asking the right questions][orientation]. They also need to get a minimum viable product deployed.
 
-For their [first milestone][milestone] they want to:
-- Ingest a representative set of simulated events.
-- Store the raw data for all events for later analysis. [:arrow_right:][cold-storage]
-- Increase the scale until they match the targets for their customer [:arrow_right:][increase-scale]
+- [Common requirements](#common-requirements)
+- [Fabrikam requirements](#fabrikam-requirements)
+- [Exploring solutions for ingestion and processing](#exploring-solutions-for-ingestion-and processing)
+- [Exploring solutions for cold storage](#exploring-solutions-for-cold-storage)
+- [Reference implementation](#reference-implementation)
+- [Lessons learned](#lessons-learned)
 
-They need a basic implementation that maps to the highlighted components of the logical architecture:
+## Common requirements
 
-![plan for this first milestone](media/01-cold-storage/logical-architecture-for-milestone01.png)
+Here are some common requirements for ingestion and cold storage that any IoT solution will at least need to consider:
+
+- Achieving scale. That means both hitting the expected target data rate, but also designing the system to be scalable in the future.
```

Comment:
Based on the conversation we had today regarding the #165 (and #193), do you think we should talk about throughtput and capcity?